### ISSUE TITLE
Invalid grant error, publishing post is impossible 

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmith/scribefire-chrome#592]
```
What's the problem?

Each time I try to publish a post, I get this error:

{ "error" : "invalid_grant" }



What browser are you using?

Firefox 11.0



What version of ScribeFire are you running?

Next 4.0
```

-----
Original issue reported on code.google.com by `g.cevol...@gmail.com` on 23 Mar 2012 at 9:52

**Comment by GoogleCodeExporter** [comment]
```
I have the same problem.  Cannot post an article to my blog.  Get the same 
error.

FF 12

SF 4.0
```

-----
Original comment by `roar...@gmail.com` on 22 May 2012 at 7:24

**Comment by GoogleCodeExporter** [comment]
```
In my case , there is something wrong with google and SF authorization. Deleted 
my blog but could not grant auth again. Deleted old auth from my google account 
by hand , tried to grant a new one withouth success. I gave auth for Sf to 
access my google account, a new windows open with auth code but it suddenly 
closes and auth is not given. I had all working fine till June. Seems some 
oddity between google and SF.
-

```

-----
Original comment by `livraria...@gmail.com` on 1 Aug 2012 at 1:29

**Comment by GoogleCodeExporter** [comment]
```
Fixed for blogger blogspot in version 4.3.1
```

-----
Original comment by `scribefi...@gmail.com` on 14 Apr 2014 at 7:08
* Changed state: **Fixed**
